pkg/api/add_events/add_events.go

```diff
@@ -66,7 +66,7 @@ type SessionInfo struct {
 }
 
 type AddEventsRequestParams struct {
-	Token       string       `json:"token,omitempty"`
+	// Token       string       `json:"token,omitempty"`
```

Comment:
Was this a testing change? Should it be removed?

Reply:
Yes, it should have been removed.